roles/kubernetes/node/templates/kubelet.rkt.service.j2

```diff
@@ -62,7 +62,8 @@ ExecStart=/usr/bin/rkt run \
                 $KUBELET_ARGS \
                 $DOCKER_SOCKET \
                 $KUBELET_REGISTER_NODE \
-                $KUBELET_NETWORK_PLUGIN
+                $KUBELET_NETWORK_PLUGIN \
+                $KUBELET_CLOUDPROVIDER
```

Comment:
Unfortunately, I was not able to find 'cloud-provider' at [rkt codebase](https://github.com/coreos/rkt/search?utf8=✓&q=cloud-provider)

Am I doing anything wrong?